back/src/main/java/ro/utcn/sd/he/assignment1/command/AddQuestionCommand.java

```diff
@@ -0,0 +1,31 @@
+package ro.utcn.sd.he.assignment1.command;
+
+import lombok.Data;
+import ro.utcn.sd.he.assignment1.model.Question;
+import ro.utcn.sd.he.assignment1.model.Tag;
+import ro.utcn.sd.he.assignment1.persistence.api.RepositoryFactory;
+
+import java.util.Optional;
+
+@Data
+public class AddQuestionCommand implements Command {
```

Comment:
Very similar with @alecostantea: just this one command implemented with the command pattern.